ui/src/containers/Layout.js

```diff
@@ -101,43 +103,52 @@ const Layout = (props) => {
 
   return (
     <ThemeProvider theme={theme}>
-      <CoreUILayout sidebar={isUserLoaded && sidebarConfig} navbarElement={<Navbar />}>
+      <CoreUILayout
+        sidebar={isUserLoaded && sidebarConfig}
+        navbarElement={<Navbar />}
+      >
         <Notifications
           notifications={notifications}
           onDismiss={removeNotification}
         />
-        <Switch>
-          <PrivateRoute exact path="/nodes/create" component={NodeCreateForm} />
-          <PrivateRoute
-            exact
-            path="/nodes/:id/deploy"
-            component={NodeDeployment}
-          />
-          <PrivateRoute
-            path={`/nodes/:id/createVolume`}
-            component={CreateVolume}
-          />
-          <PrivateRoute
-            exact
-            path="/volumes/createVolume"
-            component={CreateVolume}
-          />
-          <PrivateRoute path="/nodes" component={NodePage} />
-          <PrivateRoute exact path="/environments" component={SolutionList} />
-          <PrivateRoute path="/volumes/:name?" component={VolumePage} />
-          <PrivateRoute
-            exact
-            path="/environments/create-environment"
-            component={EnvironmentCreationForm}
-          />
-          <PrivateRoute exact path="/about" component={About} />
-          <PrivateRoute exact path="/" component={ClusterMonitoring} />
-          <PrivateRoute
-            exact
-            path="/environments/:id"
-            component={SolutionDetail}
-          />
-        </Switch>
+        <Suspense fallback={<Loader />}>
```

Comment:
May want to sync with @Cuervino on the visual design for this fallback. Anyway, I think this topic (lazy-loading pages and splitting the bundle) deserves its own PR and design.